docs/developer/extending/apps/installing-apps.mdx

```diff
@@ -166,12 +238,9 @@ The `appDeleteFailedInstallation` mutation will delete failed installation.
 The mutation takes the following fields:
 - `id`: the id of the `AppInstallation`, required.
 
+### Local Apps
 
-
-## Local Apps
-
-
-### Creating an App
+#### Creating an App
 
 To create a new app, use the `appCreate` mutation. This mutation is only available to staff users with the `MANAGE_APPS` permission. Staff user can't grant app more permissions than he has for his own. The mutation takes the following input fields:
```

Comment:
We duplicate a lot of, this part `This mutation is only available to staff users with the `MANAGE_APPS` permission. Staff user can't grant app more permissions than he has for his own.` . Maybe we can extract it somewhere

Reply:
@korycins WDYT about only mentioning it in the key concepts section? Theres already permissions part?

Reply:
Sounds good.